.babelrc

```diff
@@ -5,12 +5,10 @@
       "@babel/env",
```

Comment:
Do we even need Babel with node 20?

Reply:
I think we still need `@babel/flow` to transpile flow types.

But yes I agree we should remove it eventually, but it's probably better to do that once we migrate the code to TypeScript & esbuild.

Reply:
Yup makes perfect sense